cogs/deadline.py

```diff
@@ -84,6 +88,11 @@ async def timenow_error(self, ctx, error):
                       help="add homework and due-date $addhw CLASSNAME HW_NAME MMM DD YYYY optional(HH:MM) "
                       "ex. $addhw CSC510 HW2 SEP 25 2024 17:02")
     async def duedate(self, ctx, coursename: str, hwcount: str, *, date: str):
+        if not ctx.channel.name == 'assignments':
+            await ctx.author.send('Please send questions to the #assignments channel.')
```

Comment:
'Please send questions to the #assignments channel.'

Should be:

'Please use this command inside the #assignments channel.'